### ISSUE TITLE
about your label

### ISSUE THREAD

**Issue by HardLu** [maxboels/3D-Unet-for-Segmentation-of-Lung-lobes-in-CT-volumes#1]
hello, i can see left and right lung in your labels, whether there are more organs label, thank you!

**Comment by maxboels** [comment]
Yes, there are different types of lobes to segment and therefore different labels in the masks.

**Comment by HardLu** [comment]
> Yes, there are different types of lobes to segment and therefore different labels in the masks.

thank you, i mean is there other label in your data except right lung and left?

**Comment by sunshinesjw1** [comment]
hello, can i ask if you generated only one patch from one 256*256*256 size image?
